crates/subspace-farmer-components/benches/utils/mod.rs

```diff
@@ -1,7 +1,7 @@
 use async_trait::async_trait;
 use std::error::Error;
 use subspace_core_primitives::{Piece, PieceIndex};
-use subspace_farmer_components::plotting::PieceReceiver;
+use subspace_networking::PieceReceiver;
```

Comment:
It is extremely unfortunate that we depend on the whole `subspace-networking` just to import one trait. What if we leave this crate as is and implement its trait with trait from `subspace-networking` at `subspace-farmer` level?